boat_tutorials/conceptual/target_strength.md

```diff
@@ -1,3 +1,62 @@
-# Target Strength
+# Target strength
 
-Coming soon!
+
+## Introduction
+- intuitively, what is target strength (TS)? 
+    - how much sound energy bounces off (scatters) from a target? 
+absorption
+    - more generally: scattering strength?
+- when/how is it used?
+    - characterize scatterers or scattering sources independent of the complications of the transmit signals ([connec to SL]()) and the propagation environment ([connect to TL]())
+
+
+## Setup and terminology
+- monostatic vs bistatic ([deeper topic]())
+- differential scattering cross section --> TS ([connect to decibel page]())
+- total scattering cross section
+- extinction cross section (total power removed from the incident sound):
+    - scattering cross section + absorption cross section
+
+## Discrete scatterers
+- what can we infer using these models?
+    - fish/zooplankton abundance
+    - density of suspended sediment ([deeper topic]())
```

Comment:
- connect with inference problem
- conceptual intro here, and point to the inference class afterwards
- exercise on inference problem